filebeat/tests/system/test_modules.py

```diff
@@ -169,6 +169,7 @@ def _test_expected_events(self, test_file, objects):
 
         for ev in expected:
             found = False
+            clean_keys(ev)
```

Comment:
Should we wait till https://github.com/elastic/beats/pull/9855/files#r244956960 is merged and backported, then rebase this PR on `6.x`?

Reply:
Oh no! Totally missed the comments there. Thanks!!